.eslintignore

```diff
@@ -0,0 +1,16 @@
+assets/dist
+node_modules
+acceptance_tests
+admin_pages
+caffeinated
+core
+docs
+languages
+modules
+payment_methods
+public
+shortcodes
+tests
+ui
+widgets
+wp-assets
```

Comment:
A lot of folders excluded :)  The intent here is to only include .js files within the root folder and `assets/src` for now.  Otherwise we're going to be bombarded by js linting errors everytime we open legacy js :)